### PR TITLE
update doublet score info

### PR DESCRIPTION
**Pull request by alexvpickering**
Updating the doublet score theshold info as it was inaccurate. Here it is rendered locally:

image

### CODE REVIEW

File: src/pages/experiments/[experimentId]/data-processing/index.jsx [hunk 1]
```diff
@@ -272,7 +272,7 @@ const DataProcessingPage = ({ experimentId, experimentData, route }) => {
     {
       key: 'doubletScores',
       name: 'Doublet filter',
-      description: 'Droplets may contain more than one cell. In such cases, it is not possible to distinguish which reads came from which cell. Such “cells” cause problems in the downstream analysis as they appear as an intermediate type. “Cells” with a high probability of being a doublet should be excluded. The probability of being a doublet is calculated using ‘scDblFinder’. The cut-off is typically set around 0.5.',
+      description: <span>Droplets may contain more than one cell. In such cases, it is not possible to distinguish which reads came from which cell. Such “cells” cause problems in the downstream analysis as they appear as an intermediate type. “Cells” with a high probability of being a doublet should be excluded. The probability of being a doublet and default threshold is calculated using ‘scDblFinder’. For further information see <a href="https://bioconductor.org/packages/devel/bioc/vignettes/scDblFinder/inst/doc/scDblFinder.html#thresholding" target="_blank">here</a>.</span>,
```
Comment (aerlaut) [inline src/pages/experiments/[experimentId]/data-processing/index.jsx]: instead of `here`, maybe it would be more informative to use a more descriptive phrase, e.g. `scDblFinder threshold`?

**Comment by aerlaut** [comment]
@vickymorrison do you have any comments on the wording?

**Comment by vickymorrison** [comment]
The info box should explain what the default threshold is and how it is calculated e.g. is it a standard default for all samples or calculated based on the data.

**Comment by alexvpickering** [comment]
I changed the text @vickymorrison to try to address your comments. Unfortunately the thresholding isn't simple and I feel it's getting a bit wordy now. Happy to change if you have an alternate suggestion.

**Review by vickymorrison — APPROVED** [review]
